lib/account/send_message_helper.dart

```diff
@@ -1115,9 +1118,6 @@ class SendMessageHelper {
           .deleteByIds(pinMessageMinimals.map((e) => e.messageId).toList());
     }
 
-    _messageDao.notifyMessageInsertOrReplaced(
```

Comment:
there is needed to notify the pinned message has changed, need add/remove the pin icon when message has been pinned or unpinned.